### PR TITLE
fix(AdvancedDataCard): Removed divider in extra, added 32px spacing and modified HighlightedValueBlock

### PR DESCRIPTION
**Pull request by nicholasbrutti-tlf**
Jiras:

Advanced Data Card - Divider: [OBVIVO-2287](https://jira.tid.es/browse/OBVIVO-2287)

Highlighted Value Block - add text: [OBVIVO-2288](https://jira.tid.es/browse/OBVIVO-2288)

### CODE REVIEW

**Comment by yceballost** [comment]
> Jiras:
> 
> Advanced Data Card - Divider: [OBVIVO-2287](https://jira.tid.es/browse/OBVIVO-2287)
> 
> Highlighted Value Block - add text: [OBVIVO-2288](https://jira.tid.es/browse/OBVIVO-2288)

Link specs file to PR body, please

**Comment by nicholasbrutti-tlf** [comment]
For now this PR is on-hold for further changes.

**Comment by atabel** [comment]
> For now this PR is on-hold for further changes.

@nicholasbrutti-tlf, Is this PR ready for review?

File: src/community/advanced-data-card.tsx [hunk 1]
```diff
@@ -250,7 +251,8 @@ type AdvancedDataCardProps = MaybeTouchableCard<{
     description?: string;
     descriptionLinesMax?: number;
     extra?: ReadonlyArray<RendersNullableElement<AllowedExtra>>;
-    extraDividerPadding?: 8 | 24;
+    extraDividerPadding?: 8 | 16 | 24;
+    noDivider?: boolean;
```
Comment (atabel) [inline src/community/advanced-data-card.tsx:255]: As you already have `extraDividerPadding`, I'd call this prop `noExtraDivider`, to follow the same naming

**Review by atabel — COMMENTED** [review]
I miss some screenshot test covering the new case

File: src/community/advanced-data-card.tsx (hunk 1 above)
Comment (nicholasbrutti-tlf) [reply src/community/advanced-data-card.tsx]: Okay

**Comment by yceballost** [comment]
Specs → https://www.figma.com/design/35iS2dwoogulJZPgFGKmCX/Advanced--Data-Card?node-id=443-32508

**Review by atabel — COMMENTED** [review]
Please @nicholasbrutti-tlf, add some screenshot test covering the new case

File: src/community/advanced-data-card.css.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ export const touchableCardOverlay = style({
                 [`${touchableContainer}:hover &`]: {
                     backgroundColor: skinVars.colors.backgroundContainerHover,
                 },
-                [`${touchable}:active &`]: {
+                [`${touchableContainer}:active &`]: {
```
Comment (marcoskolodny) [inline src/community/advanced-data-card.css.ts:87]: This breaks the active state when card is interactive and you have top actions. When you click on the top action button, the entire card reacts to the click and that is not correct

File: src/community/__screenshot_tests__/advanced-data-card-screenshot-test.tsx [hunk 3]
```diff
@@ -41,6 +41,22 @@ test.each`
...
 });
 
+test.each`
+    extra
+    ${3}
+`('Advanced Data Card extras: $extra no divider', async ({extra}) => {
+    await openStoryPage({
+        id: 'community-advanceddatacard--default',
+        device: 'MOBILE_IOS',
+        args: {extra, noExtraDivider: true},
```
Comment (marcoskolodny) [inline src/community/__screenshot_tests__/advanced-data-card-screenshot-test.tsx:51]: No need for the `test.each`. Just set:

`args: {extra: 3, noExtraDivider: true},`

**Review by atabel — COMMENTED** [review]
please, @nicholasbrutti-tlf can you fix the issues commented by Marcos?

**Comment by atabel** [comment]
@nicholasbrutti-tlf, @brunocezar-tlf, please, can you fix the commented issues so we can merge this?

**Review by marcoskolodny — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**